src/module/item/SR5Item.ts

```diff
@@ -16,12 +16,19 @@ import { AdvancedRollProps, ShadowrunRoll, ShadowrunRoller } from '../rolls/Shad
 import Template from '../template';
 import { createChatData } from '../chat';
 import { SYSTEM_NAME } from '../constants';
+import ConditionData = Shadowrun.ConditionData;
+import { SR5ItemDataWrapper } from './SR5ItemDataWrapper';
 
 export class SR5Item extends Item {
     labels: {} = {};
     items: SR5Item[];
     actor: SR5Actor;
 
+    private get wrapper(): SR5ItemDataWrapper {
+        // @ts-ignore
```

Comment:
Why is this ignored?

Reply:
this was ignored because SR5Item.data is not the correct type, fixed to better show this in the update. Will eventually make the data object on SR5Item be SR5ItemType